.solhint.json

```diff
@@ -7,6 +7,7 @@
     "func-visibility": ["error", { "ignoreConstructors": true }],
     "max-line-length": ["error", 120],
     "not-rely-on-time": "off",
+    "no-global-import": "off",
```

Comment:
I think it is better to fix how we import in .sol files rather than switching off this hint